src/py42/modules/securitydata.py

```diff
@@ -196,12 +271,15 @@ def __init__(self, plan_uid, destination_guid, node_guid):
 
     @property
     def plan_uid(self):
+        """A UID for a storage plan."""
         return self._plan_uid
 
     @property
     def destination_guid(self):
+        """A GUID for the server containing the storage archive."""
```

Comment:
A "destination" is a grouping of (potentially) multiple servers that make up a broader unit, so this should be:

"A GUID for the destination containing the storage archive."